packages/paste-website/src/component-examples/ToggleExamples.ts

```diff
@@ -0,0 +1,44 @@
+export const basicToggle = `
+const ToggleExample = () => {
```

Comment:
issue: these examples aren't rendering on the site, though I'm not sure why... here's the error I get:
<img width="593" alt="image" src="https://user-images.githubusercontent.com/77117846/183766107-6629536b-aa10-4d20-8537-7ef2ff1b9533.png">

Reply:
yeah I just haven't spent much time on the docs page yet, it's still in my to-do's, I'll let ya know when I finish so you can review it! I'll keep your comments in mind while I do it 👍